src/SignalR.Orleans/Core/HubContext.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Orleans;
+using SignalR.Orleans.Clients;
+using SignalR.Orleans.Groups;
+using SignalR.Orleans.Users;
+
+namespace SignalR.Orleans.Core
+{
+    public class HubContext<THub>
+    {
+        private readonly IGrainFactory _grainFactory;
+        private readonly string _hubName;
+
+        public HubContext(IGrainFactory grainFactory)
+        {
+            _grainFactory = grainFactory;
+            var hubType = typeof(THub);
+            _hubName = hubType.IsInterface && hubType.Name.StartsWith("I")
```

Comment:
Yes, we should have support interfaces of `THub` because in the `grain project` you will only have reference to interface from the `hubs`. The implementation of the hubs will be done on the client (web project).